.github/workflows/release.yml

```diff
@@ -186,4 +176,39 @@ jobs:
         without-checkout: true
         project: maven
         custom-command: cd concordium-sdk && mvn javadoc:javadoc -f pom.xml # Generates javadoc from the delombok'ed code
-   
+    - name: Setup and maven central
+      uses: actions/setup-java@v4
+      with: # running setup-java again overwrites the settings.xml
+        distribution: 'adopt'
+        java-version: '8'
+        server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
+        server-username: MAVEN_USERNAME # env variable for username in deploy
+        server-password: MAVEN_PASSWORD # env variable for token in deploy
+        gpg-private-key: ${{ secrets.SONATYPE_PKEY }} # Value of the GPG private key to import
+        gpg-passphrase:  GPG_PASSPHRASE # Passphrase for the GPG private key
+    - name: Publish java package to central maven repository
+      run: cd concordium-sdk && mvn --batch-mode deploy -Pdeploy
+      env:
+        MAVEN_USERNAME: ${{ secrets.SONATYPE_USR }}
+        MAVEN_PASSWORD: ${{ secrets.SONATYPE_PWD }}
+        GPG_PASSPHRASE: ${{secrets.SONATYPE_PHRASE }}
+    - name: Copy javadocs and sources for android
+      run: >
+        mkdir -p concordium-android-sdk/target
+        && mv concordium-sdk/target/*-javadoc.jar concordium-android-sdk/target/concordium-android-sdk-javadoc.jar
+        && mv concordium-sdk/target/*-sources.jar concordium-android-sdk/target/concordium-android-sdk-sources.jar
+    - name: Publish android package to central maven repository
+      run: cd concordium-android-sdk && mvn --batch-mode deploy -Pdeploy
+      env:
+        MAVEN_USERNAME: ${{ secrets.SONATYPE_USR }}
+        MAVEN_PASSWORD: ${{ secrets.SONATYPE_PWD }}
+        GPG_PASSPHRASE: ${{secrets.SONATYPE_PHRASE }}
+      # Attach jar files to release (jar, jar-with-dependencies and javadoc-jar)
```

Comment:
Can you perhaps add the URLs of the deployed maven packages to the README. E.g. https://central.sonatype.com/artifact/com.concordium.sdk/concordium-android-sdk

Also, I believe we should only upload the "with-dependencies" artifact to the github release page, as the transitive dependencies cannot be resolved anyhow for the other one.

Reply:
Do we even want to continue publishing to GitHub releases now that it will be available through Maven Central?